netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -180,6 +181,18 @@ public NettyServerBuilder sslContext(SslContext sslContext) {
   }
 
   /**
+   * Sets the {@link ProtocolNegotiator} to be used. If non-{@code null}, overrides the value
+   * specified in {@link #sslContext(SslContext)}.
+   *
+   * <p>Default: {@code null}.
+   */
+  public final NettyServerBuilder protocolNegotiator(
```

Comment:
If those are the only things that use it, it should be `@Internal`. It is not intended for normal usage. `@ExperimentalApi` is just that it isn't API-stable, but people are free to use it. If you are using something `@Internal` and aren't implementing a transport "you're doing it wrong."